pkg/kv/kvserver/multiqueue/multi_queue.go

```diff
@@ -272,6 +272,9 @@ func (m *MultiQueue) QueueLen() int {
 	return m.queueLenLocked()
 }
 
+// TODO(lyang24): queueLenLocked method returns the length of the queue as if
```

Comment:
It would be much nicer if you could restructure the commits to have this change in the first commit rather than the second. Since the first commit adds the concurrencyLimit and the second removes it. If you are OK with it I'll restructure this for you this afternoon since I know git is sometimes hard to use well for rebases.